client/transactions/uncaptured/index.tsx

```diff
@@ -0,0 +1,243 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+import { TableCard, TableCardColumn, Search } from '@woocommerce/components';
+import { Button } from '@wordpress/components';
+import { onQueryChange, getQuery } from '@woocommerce/navigation';
+
+/**
+ * Internal dependencies
+ */
+import { useAuthorizations, useAuthorizationsSummary } from 'data/index';
+import Page from '../../components/page';
+import autocompleter from 'transactions/autocompleter';
+
+interface Column extends TableCardColumn {
+	key:
+		| 'authorization_id'
+		| 'authorized_on'
+		| 'captured_by'
+		| 'order'
+		| 'risk_level'
+		| 'amount'
+		| 'customer_email'
+		| 'customer_country';
+	visible?: boolean;
+	cellClassName?: string;
+}
+
+const getColumns = (): Column[] =>
+	[
+		{
+			key: 'authorization_id',
+			label: __( 'Authorization Id', 'woocommerce-payments' ),
+			visible: false,
+			isLeftAligned: true,
+		},
+		{
+			key: 'authorized_on',
+			label: __( 'Authorized on', 'woocommerce-payments' ),
+			screenReaderLabel: __( 'Authorized on', 'woocommerce-payments' ),
+			required: true,
+			isLeftAligned: true,
+			defaultOrder: 'desc',
+			cellClassName: 'date-time',
+			isSortable: true,
+			defaultSort: true,
+		},
+		{
+			key: 'captured_by',
+			label: __( 'Captured by', 'woocommerce-payments' ),
```

Comment:
Looking at the design this column should be labeled 'Capture by'

Reply:
Oops that's a typo. Fixed it